test/suites/integration/bsp/submit-proofs.test.ts

```diff
@@ -359,8 +363,14 @@ describeBspNet(
       const challengePeriod = challengePeriodResult.asOk.toNumber();
       // Then we calculate the next challenge tick.
       const nextChallengeTick = lastTickBspTwoSubmittedProof + challengePeriod;
-      // Finally, advance to the next challenge tick.
-      await userApi.advanceToBlock(nextChallengeTick);
+
+      const currentBlock = await userApi.rpc.chain.getBlock();
+      const currentBlockNumber = currentBlock.block.header.number.toNumber();
+
+      if (nextChallengeTick < currentBlockNumber) {
+        // Advance to the next challenge tick if needed
+        await userApi.advanceToBlock(nextChallengeTick);
+      }
```

Comment:
This is still the other way around, right? The condition to advance to the next challenge tick should be `nextChallengeTick > currentBlockNumber`

Reply:
yep... I am 100% sure I had fix it.